acme/tests/test_util.py

```diff
@@ -10,6 +10,7 @@
 import josepy as jose
 from OpenSSL import crypto
 import pkg_resources
+from josepy.util import ComparableECKey
```

Comment:
I think this can now be fetched via `jose` instead due to https://github.com/certbot/josepy/commit/2f33aa9d6fad66315d3cbf6a674cc54462a149fe and there's a new release of `josepy` containing this patch.

Reply:
I can't get `tools/pinning/current/repin.sh` to work; it's complaining about `poetry`, and recreating the virtual environment doesn't work.

Reply:
1. What's the specific error about Poetry?
2. `josepy` is currently pinned to 1.9.0 for other reasons: https://github.com/certbot/certbot/blob/afc5be5abeeee60560113c5eda0abfeaf1e04be8/tools/pinning/current/pyproject.toml#L54-L57. Perhaps this optimization should be left for a later time.

Reply:
```
➜ ./tools/pinning/current/repin.sh
Please install poetry 1.2+.
You may need to recreate Certbot's virtual environment and activate it.
```

And yes, I noticed `mypy` is not happy when I set it to `1.12`, so +1 to postponing this.